languages/clojure/exercises/concept/basic-functions/test/basic-functions_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns basic-functions-test
+  (:require [clojure.test :refer [deftest is]]
+            basic-functions))
+
+(deftest new-function-returns-42
+  (is (= 42 (new-function))))
+
+(deftest plus-42-adds-42
+  (is (= 0 (plus-42 42)
```

Comment:
```suggestion
  (is (= 84 (basic-functions/plus-42 42))))
```

Reply:
@bemself why do you need to explicitly write the namespace when these functions are defined in the same file right next to each other?